api/src/test/resources/application-componenttest.properties

```diff
@@ -88,3 +88,18 @@ launchDarkly.connectionTimeout=3
 launchDarkly.socketTimeout=3
 launchDarkly.flushInterval=10
 spring.datasource.hikari.maximum-pool-size=2
+
+#PCI-PAL-Antenna
+pci-pal.antenna.grant.type=${PCI_PAL_ANTENNA_GRANT_TYPE:client_credentials}
+pci-pal.antenna.tenant.name=${PCI_PAL_ANTENNA_TENANT_NAME:HMCTS}
+pci-pal.antenna.user.name=${PCI_PAL_ANTENNA_USER_NAME:APIUser}
+pci-pal.antenna.client.id=${PCI_PAL_ANTENNA_CLIENT_ID:HMCTSStage}
+pci-pal.antenna.client.secret=${PCI_PAL_ANTENNA_CLIENT_SECRET:Vm82kOUZAf9U4Kms3hIQM03jqsDQKc8aHIGy5I9y}
+pci-pal.antenna.get.tokens.url=${PCI_PAL_ANTENNA_GET_TOKENS_URL:https://pcipalstaging.cloud/api/v1/token}
+pci-pal.antenna.launch.url=${PCI_PAL_ANTENNA_LAUNCH_URL:https://euwest1.pcipalstaging.cloud/api/v1/session/303/launch}
+pci-pal.antenna.view.id.url=${PCI_PAL_ANTENNA_VIEW_ID_URL:https://euwest1.pcipalstaging.cloud/session/303/view/}
```

Comment:
I think we are hardcoding TenentId [303] as part of the urls, it would be better if we make this configurable like keeping the TenentId in App Properties and build the urls dynamically

Reply:
It won't change frequently.  Just we need to change it in 2 places. If we need to have this configurable we need to create new vaults for tenantId and change the vault value of existing one's as well in all the environments.